tests/thread_msg_bus/main.c

```diff
@@ -121,12 +154,21 @@ int main(void)
     puts("THREADS CREATED");
 
     const char hello[] = "Hello Threads!";
+    int woken;
 
     for (int id = 22; id < 25; ++id) {
-        int woken = msg_bus_post(&my_bus, id, (void*)hello);
+        woken = msg_bus_post(&my_bus, id, (void*)hello);
         printf("Posted event %d to %d threads\n", id, woken);
     }
 
+    msg_t msg = {
+        .type = 0x1337,
+        .content.ptr = "Hello everyone!",
+    };
+
+    woken = msg_send_broadcast(&msg, &my_bus);
+    printf("Posted messgage 0x%x to %d threads\n", msg.type, woken);
```

Comment:
I would suggest `Postet message 0x%x to three threads, got received by %d threads` or similar.
Maybe even assert?

Reply:
upon closer inspection I wonder if this API is even necessary - see #18762